book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
[OPTIONAL] kindly add `:rentals` accessor after `:author` because we want to access rentals in this class so kindly add it